packages/core/src/components/date-picker/date-picker.scss

```diff
@@ -89,6 +91,7 @@ $date-picker-z-index: 600;
 // ---------------------------------------------
 
 .dso-date__dialog {
+  border-width: $dso-date-dialog-border-width;
```

Comment:
Waarom is deze `border-width` nodig?

Reply:
Die stond er al op, maar nu zie je de relatie met de onderstaande berekening 
```  &.is-left {
    left: -$padding-base-horizontal + $dso-date-dialog-border-width;